### ISSUE TITLE
Inline code markup isn't very obvious

### ISSUE THREAD

**Issue by gcapes** [carpentries/workbench#97]
The old template had a shaded background in addition to the monospace font. This is similar to how things look on github e.g. `this`.

Compare with a lesson I'm currently working on which uses the new template:

![Image](https://github.com/user-attachments/assets/52643141-8620-48f2-b539-65a66fc5735d)

I imagine this will make it both harder to teach and harder to follow for learners. Could the template be updated to improve the contrast with surrounding text?

**Comment by froggleston** [comment]
Yes, this was implemented to increase contrast. Any suggestions to improve readability, but also maintain contrast for accessibility, would be welcome!

https://github.com/carpentries/workbench/issues/76

**Comment by gcapes** [comment]
I'm not really sure I understand #76. It looks like the background colour was set to white to improve contrast (of the inline code text against the background?) but this makes the distinction between inline code and normal text very subtle, so it's traded readability for accessibility.

I would suggest returning it to a shaded background of some kind or other so that it's easier to distinguish from the normal text. Or at least with the outline of a box even if the background is still white. Could the colour of the font be changed like Toby suggested? There didn't look to be a follow up on that particular suggestion. 
What about a yellow background with black text, or invert the colours so there's a black background with white text?

**Comment by gcapes** [comment]
Or as another thought - can the background colour for inline code be the same light grey as we have for the code blocks? If that's ok for accessibility it should be ok for inline code too?

**Comment by froggleston** [comment]
I've attempted to solve this issue by using slightly adjusted CSS for inline code blocks in both light and dark mode, and added a mouse over hover to help with readability. Let me know if this looks better!

![Image](https://github.com/user-attachments/assets/d887983b-b42d-4be2-bc32-f025855e8d6e)

**Comment by ErinBecker** [comment]
Thanks for implementing this @froggleston! This looks great to me. I'll wait for @gcapes to confirm if this looks good to them and then we can merge.

**Comment by gcapes** [comment]
That is an improvement, thanks. For me though,  I don't think there's any need for the hover as it suggests you can click on it for more info (which you can't). I do prefer that background colour that you've used for the hover as there's more contrast with the standard text. Could you use that/those background colours instead and remove the hover?

**Comment by froggleston** [comment]
I think I intended the hovering to be useful when commands span multiple lines, so it's clearer for where the command or info starts and stops. The arrow doesn't change to the clickable "finger" so I would hope it was fairly apparent it was only for display.

Thanks for the feedback though - I'll take this to the maintainers to get some wider comments!

**Comment by ndporter** [comment]
I agree that the looks are nicer. I'm mixed on the hover behavior. @gcapes is right that conventionally things that change with hover are clickable, but I do appreciate the ability to highlight the range of the inline code snippet. Hopefully some others on `maintainers` will chime in.

**Comment by froggleston** [comment]
@gcapes @ndporter So the consensus is to remove the hover? I'd like to get a release done this week so happy to remove and revisit at a later date!

**Comment by gcapes** [comment]
Yep, my view is to remove the hover, and use the darker background colours from the hover instead of the lighter non-hover colours.
Thanks for looking into this.

Next question, is how do I get these changes into my own lesson? Do I need to add this repo as an upstream remote, or does one of the workflows do some magic?

**Comment by gcapes** [comment]
Great, thanks for this (https://github.com/carpentries/varnish/pull/172). 

> Next question, is how do I get these changes into my own lesson? Do I need to add this repo as an upstream remote, or does one of the workflows do some magic?

**Comment by gcapes** [comment]
Hi, 
I see this has been fixed via https://github.com/carpentries/varnish/pull/172. Does that mean this issue should be closed?

Also, I'm not clear how to incorporate these changes into my own lesson (in development). Do  I need to pull changes from an upstream remote, or does one of the github workflows do the required steps for me?

Thanks

**Comment by froggleston** [comment]
Hey @gcapes - so sorry for missing your replies there. Yes, we can close this issue if you feel that it addresses your original points?

The changes should be pulled in when you run your Build and Deploy workflow runs. You can check this by looking at the `sessionInfo()` output under the Setup Lesson Engine step, e.g. https://github.com/swcarpentry/shell-novice/actions/runs/17389844471/job/49361864110#step:5:576

**Comment by gcapes** [comment]
Great, thanks. If these changes are in a new release of varnish(?), then it's done the job for me. I'll check my next build - which version number should I be looking for? 
Thanks

**Comment by froggleston** [comment]
Hey @gcapes - the latest varnish version is 1.0.7 so as long as you're seeing that in your builds then you should be good! I'll close this issue, but please do reopen/raise another if there are further improvements!

**Comment by gcapes** [comment]
Thanks, I have varnish `1.0.7` , but the inline code rendering doesn't appear to show any different background colour to the rest of the text: 

<img width="782" height="301" alt="Image" src="https://github.com/user-attachments/assets/ea05201b-eec0-4432-b7c8-d10dc78c7138" />

**Comment by froggleston** [comment]
Hi @gcapes - could you post your sessionInfo() details please? I'm using the latest release versions and I can see the changed backgrounds.

**Comment by gcapes** [comment]
I think this might have just taken a while to deploy. It looks ok now. Thanks!

<img width="253" height="40" alt="Image" src="https://github.com/user-attachments/assets/cfb9d6a5-7c48-406a-9f40-65d2e693788b" />

**Comment by froggleston** [comment]
Awesome! 🥳
